docs/index.html

```diff
@@ -0,0 +1,445 @@
+        </div>
+            </div>
+        </div>
+    </div>
+
+
+    <section id="about" class="page-section">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-3 col-sm-6 col-xs-12">
+                    <div class="service-item">
+                        <div class="icon">
+                            <img src="img/service_icon_01.png" alt="">
+                        </div>
+                        <h4>Simple and Lightweght</h4>
+                        <div class="line-dec"></div>
+                        <p>&nbsp;A set of lightweight modules with a framework that strongly encourages convention over configuration, while highlighting the criticality of structure.</p>
+                        <div class="primary-blue-button">
+                            <a href="#" class="scroll-link" data-id="portfolio">Continue Reading</a>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-3 col-sm-6 col-xs-12">
+                    <div class="service-item">
+                        <div class="icon">
+                            <img src="img/service_icon_02.png" alt="">
+                        </div>
+                        <h4>Couchbase Library</h4>
+                        <div class="line-dec"></div>
+                        <p>Provides a couchbase library to build dynamic queries based on Spring Data and Query DSL</p>
+                        <div class="primary-blue-button">
+                            <a href="#" class="scroll-link" data-id="portfolio">Continue Reading</a>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-3 col-sm-6 col-xs-12">
+                    <div class="service-item">
+                        <div class="icon">
+                            <img src="img/service_icon_03.png" alt="">
+                        </div>
+                        <h4>Rapid Development</h4>
+                        <div class="line-dec"></div>
+                        <p>&nbsp;Designed with the purpose of helping developers create web services in a quick and easy way following strict conventions.</p>
+                        <div class="primary-blue-button">
+                            <a href="#" class="scroll-link" data-id="portfolio">Continue Reading</a>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-3 col-sm-6 col-xs-12">
+                    <div class="service-item">
+                        <div class="icon">
+                            <img src="img/service_icon_04.png" alt="">
+                        </div>
+                        <h4>High Quality Built-In</h4>
+                        <div class="line-dec"></div>
+                        <p>Covers a majority of the scenarios you will need when creating an enterprise-grade application, it is also very open to extension.</p>
```

Comment:
I would either change the comma splice from "application, it is" to "application. It is " or just delete the "it is also very open to extension" part

Reply:
done